### PR TITLE
refactor: button styles

### PR DESCRIPTION
**Pull request by patricio0312rev**
# [[general] refactor button styles](https://app.clickup.com/t/86dug4xy7)

## Summary

- Button props for `roundedClassName` and `sizeClassName` have been removed.
- Implementations of these props have been refactored to simply use `className`.
- Snapshots have been updated.

## Checklist

-   [ ] My changes look good in both light AND dark mode
-   [ ] The change is not hardcoded to a single network, but has multi-asset in mind
-   [ ] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: src/domains/wallet/pages/CreateWallet/SuccessStep.tsx [hunk 1]
```diff
@@ -72,8 +72,7 @@ export const SuccessStep = ({ onClickEditAlias }: { onClickEditAlias: () => void
 								data-testid="CreateWallet__edit-alias"
 								type="button"
 								variant="transparent"
-								className="text-theme-navy-600"
-								sizeClassName="py-0"
+								className="space-x-0 px-0 py-0 text-theme-navy-600"
```
Comment (shahin-hq) [inline src/domains/wallet/pages/CreateWallet/SuccessStep.tsx:75]: there a few additional classes, is this expected?

**Review by shahin-hq — COMMENTED** [review]
Looks good, I'll approve once you reply the comments.

File: src/domains/wallet/pages/CreateWallet/SuccessStep.tsx (hunk 1 above)
Comment (patricio0312rev) [reply src/domains/wallet/pages/CreateWallet/SuccessStep.tsx:75]: Yes, so I added them to make sure the default styles (`px-5 py-3 space-x-3 text-base`) don't affect this implementation

File: src/domains/contact/components/ContactForm/ContactForm.blocks.tsx [hunk 2]
```diff
@@ -37,8 +37,7 @@ const AddressListItem: React.VFC<AddressListItemProperties> = ({ address, onRemo
 					<Button
 						data-testid="contact-form__remove-address-btn-xs"
 						size="icon"
-						sizeClassName="p-0"
-						className="text-theme-secondary-700 dark:text-theme-secondary-500 sm:!hidden"
+						className="p-0 text-theme-secondary-700 dark:text-theme-secondary-500 sm:!hidden"
```
[the remark this answers is not in the document]
Comment (patricio0312rev) [reply src/domains/contact/components/ContactForm/ContactForm.blocks.tsx:40]: Nice catch 🔥

**Comment by patricio0312rev** [comment]
@shahin-hq should be fixed now 🔥

**Comment by shahin-hq** [comment]
I'd be careful with components with extra added classes, it might break something quitely:

image

**Comment by samharperpittam** [comment]
Moved to draft due to the issue @shahin-hq  mentioned above.

**Review by ItsANameToo — APPROVED** [review]

**Review by samharperpittam — APPROVED** [review]

**Pull request merged**